.github/workflows/tag_commits_to_master.yml

```diff
@@ -0,0 +1,18 @@
+name: Tag Commits to Master
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+      with:
+        fetch-depth: '0'
+    - name: Bump version and push tag
+      uses: anothrNick/github-tag-action@1.19.0

```

Comment:
i'd prefer to use `K-Phoen/semver-release-action` It allows you to designate which branch to listen to merges on. this action requires `master` branch. see https://github.com/SU-SWS/react_paragraphs/blob/8.x-1.x/.github/workflows/release.yml

Reply:
I've changed this to your preference.